_source/_posts/2022-01-27-build-website-with-eleventy.md

```diff
@@ -0,0 +1,334 @@
+
+There's a comment to replace the code here with an API call. You can add an API call to verify the ID token on the server-side before displaying any content to the user.
+
+Finally, add a page for `README.md` and add the following markdown to it.
+
+```markdown
+---
+layout: _layout_unauthenticated.11ty.js
+---
+
+# Okta Eleventy Website Example
+
+This repository shows you how to use Okta on a static site built with 11ty.
+
+**Prerequisites:**
+
+- [Node.js 10 or newer](https://nodejs.org/en/download/)
+- [Okta CLI](https://cli.okta.com)
+```
+
+This is a truncated version of the [README file](https://github.com/oktadev/okta-website-eleventy-example/blob/master/README.md) that you can see in the GitHub repository for this project.
+
+## Add Layout Pages
+
+It's now time to add the layout pages for your static files. 11ty enables you to write layouts in several templating languages. For this project, you will use the stock `11ty.js` format. Create a new folder called `_includes` using the command `mkdir _includes`. 
+
+Add a file called `_layout_base.11ty.js` and add the following code to it.
+
+{% raw %}
+```javascript
+exports.data = {
+
+};
+
+exports.render = function (data) {
+  return `<!doctype html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
+        crossorigin="anonymous"></script>
+    <script src="https://global.oktacdn.com/okta-signin-widget/5.14.1/js/okta-sign-in.min.js"
+        type="text/javascript"></script>
+    <link href="https://global.oktacdn.com/okta-signin-widget/5.14.1/css/okta-sign-in.min.css" type="text/css"
+        rel="stylesheet" />
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>${data.title}</title>
+</head>
+
+<body>
+
+    <script type="text/javascript">
+        const oktaSignIn=new OktaSignIn({
+            baseUrl: "https://{{yourOktaDomain}}",
+            redirectUri: 'http://localhost:8080/callback',
+            clientId: "{{yourClientId}}",
+            authParams: {
+                issuer: "https://{{yourOktaDomain}}/oauth2/default"
+            }
+        });     
+    </script>
+    <div class="container">
+        ${data.content}
+    </div>
+
+</body>
+
+</html>`;
+};
+```
+{% endraw %}
+
+Be sure to replace the placeholder variables with your actual Okta information.
+
+This file will be the base layout for all of your pages.  This file loads `bootstrap` into all of your pages.  It also initializes the `OktaSignIn` widget on each page.  This way, you can access it in the javascript of your file no matter where you are.
```

Comment:
```suggestion
This file will be the base layout for all of your pages.  It loads `bootstrap` into all of your pages and also initializes the `OktaSignIn` widget on each page.  In this way, you can access it in the JavaScript of your file no matter where you are.
```
avoids the repetltion of "This" to open 3 sentences in a row, and varies the sentence length